### PR TITLE
Add cms_link fuzz target

### PR DESCRIPTION
**Pull request by Avgor46**

### CODE REVIEW

**Review by kobrineli — APPROVED** [review]

**Review by kobrineli — CHANGES_REQUESTED** [review]
Modify corpus for the new target
Add 3 bytes in the beginning of each file

File: projects/lcms/cms_link_sydr.c [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  for (i = 0; i < nargs; i++) {
+
+    if (Profiles[i] != NULL)
+      cmsCloseProfile(Profiles[i]);
+  }
+
+  return 0;
+}
+
+int main(int argc, char **argv) {
```
Comment (apach301) [inline projects/lcms/cms_link_sydr.c:149]: Why can't we just link main.o for sydr/cov fuzztargets?

**Review by PaDarochek — APPROVED** [review]

File: projects/lcms/README.md [hunk 2]
```diff
@@ -52,3 +52,7 @@ Little CMS project has 3 fuzz targets.
 ### cms_transform
 
     # sydr-fuzz -c cms_transform.toml run
+
+### cms_link
```
[the remark this answers is not in the document]
Comment (Avgor46) [reply projects/lcms/README.md:56]: done

File: projects/lcms/cms_link_sydr.c (hunk 1 above)
Comment (Avgor46) [reply projects/lcms/cms_link_sydr.c]: All sydr targets have been changed, done

**Pull request merged**